test/modules/EdgeHubRestartTester/src/Program.cs

```diff
@@ -61,17 +61,23 @@ static async Task<int> MainAsync()
 
                     // Setup Message Task
                     Task sendMessageTask = Task.CompletedTask;
-                    sendMessageTask = edgeHubMessageConnector?.StartAsync(
-                        eachTestExpirationTime,
-                        restartTime,
-                        cts.Token);
+                    if (edgeHubMessageConnector != null)
```

Comment:
Just FYI; you can also do this.

sendMessageTask = edgeHubMessageConnector?.StartAsync(eachTestExpirationTime, restartTime, cts.Token) ?? Task.Completed;

Reply:
Yeah. I was thinking about that but it's not as quite readable as explicit `if`. Thanks for the info though.